src/routes/docs/apis/rest/+page.markdoc

```diff
@@ -267,77 +262,96 @@ Appwrite's SDKs have a helper `ID.unique()` to generate unique IDs. When using A
 
 # Queries {% #queries %}
 
-Appwrite's SDKs provide a `Query` class to generate query strings. When using Appwrite without an SDK, you can template your own strings with the format below.
+Appwrite's SDKs provide a `Query` class to generate JSON query strings.
+When using Appwrite without an SDK, you can template your own JSON strings.
+You can discover the query methods available in the [Queries page.](/docs/products/databases/queries)
 
-Query strings are passed to Appwrite using the `queries` parameter. You can attach multiple query strings by including the array parameter multiple times in the query string: `queries[]="..."&queries[]="..."`
```

Comment:
The query string param part is still really helpful. Making it even more clear would actually be more helpful. For example, the query string **before** URL encoding would be

```
?queries[0]={"method":"equal","attribute":"name","values":["John"]}&queries[1]={"method":"limit","values":[6]}
```

Reply:
Added back